mobile/lib/modules/album/views/select_user_for_sharing_page.dart

```diff
@@ -13,6 +13,7 @@ import 'package:immich_mobile/shared/models/asset.dart';
 import 'package:immich_mobile/shared/models/user.dart';
 import 'package:immich_mobile/shared/ui/user_circle_avatar.dart';
 
+@RoutePage<List<String>>()
```

Comment:
Why do we need to include the type for this decorator?

Reply:
It is the used to specify the return type of the route. Previously, we used to have this directly in the `CustomRoute` class in the list of routes in `router.dart` file. This is changed now and we are to specify them in each of the page directly in the `RoutePage` annotation. This will generate the proper typed `AutoRoutePage` in the `router.gr.dart` file